contracts/eth/nearbridge/contracts/AdminControlled.sol

```diff
@@ -41,6 +42,33 @@ contract AdminControlled {
         }
     }
 
+    function verifyAdminAddress(address newAdmin) internal view onlyAdmin {
+        require(newAdmin != admin, "Nominated admin is the same as the current");
```

Comment:
`verifyAdminAddress` can be converted into a `modifier` ? e.g `onlyValidAdminAddress`

Reply:
I don't think it makes any sense, the method could be called by anyone, but it's about the verification of input parameters.